packages/kernel-6.1/kernel-6.1.spec

```diff
@@ -16,6 +16,8 @@ Source3: gpgkey-00FA2C1079260870A76D2C285749CAD8646D9185.asc
 Source100: config-bottlerocket
 Source101: config-full-bottlerocket-x86_64
 Source102: config-full-bottlerocket-aarch64
+Source103: config-bottlerocket-x86_64
+Source104: config-bottlerocket-aarch64
```

Comment:
nit: let's number these 101 and 102, and move the full configs to 200 + 201, as part of this change.

That way the parts are grouped together.